Resources/doc/getting_started/the_form_view.rst

```diff
@@ -0,0 +1,272 @@
+            ))
+        ;
+    }
+
+.. image:: ../images/getting_started_sonata_model_type.png
+
+Using Groups
+------------
+
+Currently, everything is put into one block. Since the form only has three
+fields, it is still usable, but it can become quite a mess pretty quick. To
+solve this, the form mapper also supports grouping fields together.
+
+For instance, the title and body fields can belong to the Content group and the
+category field to a Meta data group. To do this, use the ``with()`` method:
+
+.. code-block:: php
+
+    // src/AppBundle/Admin/BlogPostAdmin.php
+
+    // ...
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->with('Content')
+                ->add('title', 'text')
+                ->add('body', 'textarea')
+            ->end()
+
+            ->with('Meta data')
+                ->add('category', 'sonata_type_model', array(
+                    'class' => 'AppBundle\Entity\Category',
+                    'property' => 'name',
+                ))
+            ->end()
+        ;
+    }
+
+The first argument is the name/label of the group and the second argument is an
+array of options. For instance, you can pass HTML classes to the group in
+order to tweak the styling:
+
+.. code-block:: php
+
+    // src/AppBundle/Admin/BlogPostAdmin.php
+
+    // ...
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->with('Content', array('class' => 'col-md-9'))
+                // ...
+            ->end()
+            ->with('Meta data', array('class' => 'col-md-3')
+                // ...
+            ->end()
+        ;
+    }
+
+This will now result in a much nicer edit page:
+
+.. image:: ../images/getting_started_post_edit_grid.png
+
+Using Tabs
+~~~~~~~~~~
+
+If you get even more options, you can also use multiple tabs by using the
+``tab()`` shortcut method:
+
+.. code-block:: php
+
+    $formMapper
+        ->tab('Post')
+            ->group('Content', ...)
+                // ...
+            ->end()
+            // ...
+        ->end()
+
+        ->tab('Publish Options')
+            // ...
+        ->end()
+    ;
+
+Creating a Blog Post
+--------------------
+
+You've now finished your nice form view for the ``BlogPost`` model. Now it's
+time to test it out by creating a post.
+
+After pressing the "Create" button, you probably see a green message like:
+*Item "AppBundle\Entity\BlogPost:00000000192ba93c000000001b786396" has been
+successfully created.*
+
+While it's very friendly of the SonataAdminBundle to notify the admin of a
+successful creation, the classname and some sort of hash aren't really nice to
+read. This is the default string representation of an object in the
+SonataAdminBundle. You can change it by defining a ``toString()`` (note: no
+underscore prefix) method in the Admin class. This receives the object to
+transform to a string as the first parameter:
```

Comment:
we should hint here, that you can also use the "__toString()" from the entity itself, which is much easier for beginners.

What do you think @WouterJ